### PR TITLE
New package: ZeroTierOne

### PR DESCRIPTION
**Pull request by sjmackenzie**

### CODE REVIEW

File: pkgs/tools/networking/zerotierone/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  '';
+
+  buildInputs = [  openssl lzo zlib gcc];
+
+  installPhase = ''
+    installBin zerotier-one
+  '';
+
+  meta = {
+    description = "Create flat virtual Ethernet networks of almost unlimited size.";
```
Comment (jagajaga) [inline pkgs/tools/networking/zerotierone/default.nix]: Remove `.` at the end.

Comment (sjmackenzie) [inline pkgs/tools/networking/zerotierone/default.nix]: -.

File: pkgs/tools/networking/zerotierone/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, openssl, lzo, zlib, gcc}:
```
Comment (ryantm) [inline pkgs/tools/networking/zerotierone/default.nix]: missing space after gcc and same below on line 21

Comment (sjmackenzie) [inline pkgs/tools/networking/zerotierone/default.nix]: -

**Pull request merged**